src/pages/registration/SummaryBox.tsx

```diff
@@ -97,7 +97,7 @@ const SummaryBox = ({
                 e.preventDefault();
                 setCurrentFormIndex(tissueIndex);
               }}
-              key={tissue.clientId}
+              key={tissueIndex}
```

Comment:
better option for key than an index?

Reply:
Again , same as before there is not a unique id field exists for a given data structure